src/migrations/versions/c98da12d76f8_add_audit_log.py

```diff
@@ -0,0 +1,65 @@
+"""add audit log
+
+Revision ID: c98da12d76f8
+Revises: 90070db16d05
+Create Date: 2022-05-04 20:16:39.246537
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "c98da12d76f8"
+down_revision = "90070db16d05"
```

Comment:
heads up to keep an eye on https://github.com/ethyca/fidesops/pull/453/files#diff-d2624a58797f4cdb6db005c824414a9f1527dcdcbdcdcb95127605917fc7c115R16, which has a conflicting down_revision

Reply:
Good catch. Updated

Reply:
Oh, was just saying that depending on which one gets merged first, one of us will need to update the `down_revision` to reference the `revision` of what was last merged. Since my PR hasn't been merged yet, there aren't any changes needed here. Mind reverting to what you had for now?

Reply:
ahh gotcha, so many migrations to keep track of recently!